ocs_ci/ocs/node.py

```diff
@@ -2693,3 +2693,26 @@ def generate_nodes_for_provider_worker_node_tests():
     generated_nodes = get_node_objs(node_choice_names)
     log.info(f"Generated nodes for provider node tests: {node_choice_names}")
     return generated_nodes
+
+
+def gracefully_reboot_nodes():
+    """
+
+    Gracefully reboot OpenShift Container Platform nodes
+
+    """
+    from ocs_ci.ocs import platform_nodes
```

Comment:
pls move the import to the top of the file

Reply:
Was facing cyclic import error, so to avoid it imported inside the function